package.json

```diff
@@ -42,6 +42,7 @@
     "@types/express": "^4.17.15",
     "@types/express-fileupload": "^1.4.1",
     "@types/node": "^18.11.18",
+    "@types/express-fileupload": "^1.4.1",
```

Comment:
We already have this dependency (two lines above), why is there a duplicate lolz